benchmark/infrastructure/BenchmarkMain.cpp

```diff
@@ -120,23 +120,25 @@ int main(int argc, char** argv) {
   }
 
   // Pass the configuration, even if it is empty.
-  BenchmarkRegister::passConfigurationToAllRegisteredBenchmarks(config);
+  ad_benchmark::BenchmarkRegister::passConfigurationToAllRegisteredBenchmarks(
```

Comment:
In the `BenchmarkMain.cpp` you can maybe afford the `using namespace ad_benchmark` to make the code more readable.
(In general, `using namespace` in a `.cpp` file can be okay if it really helps, just don't do this in a header file at global scope.

Reply:
Done, see [be8c9b3](https://github.com/ad-freiburg/qlever/pull/860/commits/be8c9b3a1fd63920bd994a5277f425dac6f468c1).